packages/@aws-cdk-containers/ecs-service-extensions/test/integ.assign-public-ip.expected.json

```diff
@@ -779,6 +779,7 @@
                 "dynamodb:Query",
                 "dynamodb:GetItem",
                 "dynamodb:Scan",
+                "dynamodb:ConditionCheckItem",
```

Comment:
Do we need this action? Just want to make sure we're not adding more permissions than we actually need.